xarray/tests/test_accessor_dt.py

```diff
@@ -74,7 +76,18 @@ def test_field_access(self, field) -> None:
         else:
             data = getattr(self.times, field)
 
-        expected = xr.DataArray(data, name=field, coords=[self.times], dims=["time"])
+        if data.dtype.kind != "b" and field not in ("date", "time"):
+            # pandas 2.0 returns int32 for integer fields now
+            data = data.astype("int64")
```

Comment:
once again, `pandas` returns `int32` instead of `int64` for datetime fields now. We might want to follow that, but I didn't want to make that change without discussion.

Reply:
preserving our return dtype seems OK to me.